packages/riipen-ui/src/components/Button.jsx

```diff
@@ -165,19 +165,24 @@ class Button extends React.Component {
           }
 
           a.root {
-            min-width: 70px;
+            min-width: 100px;
             text-decoration: none;
           }
 
+          .icon {
+            font-size: 16px;
+            height: 13px;
+            margin-top: -3px;
```

Comment:
Are you sure this is right? Do font awesome icons have white space at the top of them? Super frustrating if true.

Reply:
They dont, but im trying to keep the icons the same size as the text font size. If the icons are bigger then the buttons grow in different ways ie. if we have an icon before the text the button grows upwards, while if we have an icon after the text the button grows down. this is the only way i could make them line up (tried a bunch of flex stuff but nothing seemed to help). Maybe we just live with the buttons being slightly different and handle it in the container?

Reply:
given that whatever is in the button can change size as well

Reply:
Ah ok that makes sense. Nah I think this is fine rather than trying anything else.